highlighters/main/test-data/alias-precommand-option-argument2.zsh

```diff
@@ -28,15 +28,14 @@
 # vim: ft=zsh sw=2 ts=2 et
 # -------------------------------------------------------------------------------------------------
 
-alias sde='sudo -e'
-alias seu='sde -u'
+alias sdb='sudo -b'
+alias sbu='sdb -u'
```

Comment:
Those cryptic names are a bit hard to follow. Not a big issue but could you use longer names in the future?

Reply:
Fixed.